### ISSUE TITLE
Incorrect schema version in Syft JSON format ID

### ISSUE THREAD

**Issue by fg-j** [anchore/syft#1273]
As far as I can tell, Syft v0.59.0 generates Syft JSON v4.1.0 SBOMs. But the ID of the associated format is [still `syft-3-json`](https://github.com/anchore/syft/blob/d3ee24017ec6c14cc288610680bc056520c2d541/syft/formats/syftjson/format.go#L7). I understood the ID to be referring to the major version of the schema; is that the intent? If so, this seems like a mistake.

**Comment by kzantow** [comment]
Yes, you are right @fg-j, this should be updated -- however, you can also just use `syft-json` (or `json`) for "the latest Syft JSON version".

**Comment by spiffcs** [comment]
I linked #1274 to this - I'll write a quick test so we don't go out of sync again

**Comment by fg-j** [comment]
@kzantow for some context, the tool I work on consumes syft (and its formats) as a library, so we tend to care about the value of the exported [syft.JSONFormatID](https://github.com/anchore/syft/blob/6e764815d0cb9d269d508524cdeac121184ec38f/syft/formats.go#L21).
